distributed-commandbus-springcloud/pom.xml

```diff
@@ -71,7 +79,13 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
-
+    <repositories>
```

Comment:
I've discussed this with @abuijze, and I think this is the main reason why we're not approving/merging this yet.
We'd prefer if would not have to use a milestone release of Spring Cloud. 
Do you have a clue when they'll be finalizing version 2 of Spring Cloud? 
And, do we need the Spring Cloud version to be 2.x to be able to use Spring Boot 2.x?

Reply:
Thanks, Spring cloud 2.0 will be fully compatible with Spring Boot 2.x. AFAIK, they are targeting mid April.

Reply:
@smcvb 
> And, do we need the Spring Cloud version to be 2.x to be able to use Spring Boot 2.x?

Yeah, in order to avoid 
`Caused by: java.lang.ClassNotFoundException: org.springframework.boot.bind.RelaxedDataBinder`

while executing `TrackingEventProcessorIntegrationTest` 
see https://github.com/spring-projects/spring-boot/issues/9070#issuecomment-298828725 for more details

Reply:
Ok, that's good to know!